### DISCUSSION TITLE
Vom CUxD-Login ausgesperrt

### DISCUSSION THREAD

**Discussion by calllib** [OpenCCU/OpenCCU#1634] [🇩🇪 Fragen&Antworten] [answered]
Nach ein paar Jahren Abstinenz wollte ich ein neues enOcean-Gerät am CUxD 2.6 registrieren, um leider festzustellen, dass ich ausgesperrt bin. In der Datei `/usr/local/addons/cuxd/cuxd.ini` findet sich ein Eintrag nach dem Schema `USERLOGIN=admin:pass`, der aber nicht funktioniert. Anders als die Anleitung nahelegt, gibt es im selben Verzeichnis auch keine Datei `cuxd.pwd`, auch nach einem Reboot nicht.
Wenn ich den USERLOGIN-Eintrag mit einer Zeile `USERACCESS=1` ergänze, geht der Login auch nicht.
Wenn ich ihn durch `USERACCESS=` ersetze, geht der Login (mit einem Administrations-Account) auch nicht.
Was tun?

**Comment by Baxxy13** [comment]
Vielleicht hilft dir [> das <](https://homematic-forum.de/forum/viewtopic.php?f=37&t=62715&p=619633&hilit=cuxd+ausgesperrt#p619649)

**Reply by calllib** [reply]
Danke für den Verweis. Ich hatte den Thread in dem Unterforum nicht gesehen.
Nach dem Löschen von cuxd.ini und Neustart der CCU finden sich in der automatisch neu angelegten Datei die beiden leeren Einträge
```
USERLOGIN=
USERACCESS=
```
- Wenn ich (als Admin-User) ich von der RaspberryMatic aus (Startseite > Einstellungen > Systemsteuerung) oder direkt über `http://<AdresseDerCCU>/addons/cuxd/` den CUxD ansteuere, bekomme ich ein Popup-Fenster, das mich nach Username und Password fragt. Der RaspberryMatic-Admin-Account funktioniert hier nicht, Leereingabe ebensowenig.
- Selbst wenn ich `USERACCESS=1+` setze, kommt (nach einem Neustart des CUxD) dieses Popup-Fenster, obwohl ich es nach der Anleitung nun gar nicht erwarten würde.

Was läuft da schief?

**Reply by Baxxy13** [reply]
Da musst du mal Uwe von CUxD direkt im Homematic-Forum fragen. Hier bist du ja bei RaspberryMatic.
Soweit ich weiß sollte das Öffnen der CUxD-Statusseite (über den Button der WebUI-Systemsteuerung) auch funktionieren wenn...
```
USERLOGIN=
USERACCESS=
```

nicht gesetzt sind.

**Reply by calllib** [reply]
Ich habe dort jetzt auch diese Frage eingestellt, bin mir aber nicht sicher, dass das Problem im CUxD liegt. Ich habe mittlerweile den Verdacht, dass der Fehler an anderer Stelle liegt, denn das gleiche Login-Fenster kommt manchmal auch sofort, wenn ich in der RaspberryMatic auf Startseite > Einstellungen > Systemsteuerung gehe:
![Screenshot 11](https://user-images.githubusercontent.com/16517367/147858176-4e1424e4-4571-44a8-ab86-fad6e00327ab.png)

Es fällt auch auf, dass das das CUxD-Icon nicht angezeigt wird, obwohl es unter /usr/local/addons/cuxd/CUXD.PNG mit Leserechten für alle liegt:
![Screenshot2](https://user-images.githubusercontent.com/16517367/147858247-7ad736ad-f43b-40b6-b336-fe47c17fee53.png)
Kann es sein, dass etwas global mit den Zugriffsrechten des Webservers auf das CUxD-Verzeichnis nicht stimmt? Wo müsste ich was korrekterweise konfiguriert vorfinden?

**Reply by calllib** [reply]
P.S. Ein gutes Neues!

**Comment by calllib** [comment, marked as answer]
Es scheint sich nicht um ein Problem des CUxD zu handeln, sondern allgemeiner zu sein. Deswegen unter anderem Titel neu aufgesetzt:
[Zusatzsoftware nur mit zusätzlichem Login sichtbar?](1639)

**Reply by calllib** [reply]
Lösung siehe dort.
